### PR TITLE
Team

### PR DESCRIPTION
**Pull request by birdglove2**
add team interface
member -> isMember table

### CODE REVIEW

**Review by palsp — APPROVED** [review]

File: users/src/routes/__test__/user_test/manageUserStatus.test.ts [hunk 1]
```diff
@@ -20,43 +18,39 @@ const setup = async () => {
   return { sender, receiver };
 };
 
-// should check all IF-STATEMENT cases on /api/users/invitaion as well
+// should check all IF-STATEMENT cases on /api/users/status/invitaion as well
```
[the remark this answers is not in the document]
Comment (birdglove2) [reply users/src/routes/__test__/user_test/manageUserStatus.test.ts:21]: kuy rai

File: users/src/routes/__test__/team_test/new-team.test.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import request from 'supertest';
 import { app } from '../../../app';
-import { Member } from '../../../models/member.model';
+import { IsMember } from '../../../models/isMember.model';
```
[the remark this answers is not in the document]
Comment (birdglove2) [reply users/src/routes/__test__/team_test/new-team.test.ts]: done

File: users/src/routes/__test__/team_test/manageMemberStatus.test.ts [hunk 3]
```diff
@@ -36,21 +34,21 @@ describe('Status Changing Test', () => {
   it('should return 400 if team is not found', async () => {
```
[the remark this answers is not in the document]
Comment (birdglove2) [reply users/src/routes/__test__/team_test/manageMemberStatus.test.ts]: done

File: users/src/routes/__test__/team_test/manageMemberStatus.test.ts [hunk 4]
```diff
@@ -1,32 +1,30 @@
...
-import { User } from '../../models/user.model';
+import { app } from '../../../app';
+
 import { Business } from '@cuconnex/common';
-import { TeamStatus } from '@cuconnex/common';
-import { Interest } from '../../models/interest.model';
+import { User, Interest } from '../../../models';
 
 describe('Status Changing Test', () => {
   it('should return 400 if user is not found', async () => {
```
[the remark this answers is not in the document]
Comment (birdglove2) [reply users/src/routes/__test__/team_test/manageMemberStatus.test.ts]: done

File: users/src/controllers/user.controller.ts [hunk 5]
```diff
@@ -1,155 +1,217 @@
...
     }
 
-    const relation = await req.user!.findRelation(req.params.userId)
+    res.status(200).send({ message: 'Invitation(s) is/are from these teams.', teams: teams });
+  } catch (err) {
+    throw new BadRequestError(err.message);
+  }
+};
+
+export const getListofTeamsBelongsTo = async (req: Request, res: Response) => {
```
[the remark this answers is not in the document]
Comment (birdglove2) [reply users/src/controllers/user.controller.ts:177]: done

File: users/src/controllers/team.controller.ts [hunk 6]
```diff
@@ -1,6 +1,8 @@
 import { Request, Response } from 'express';
 import { BadRequestError } from '@cuconnex/common';
-import { Team, Member, User } from '../models';
+import { Team, IsMember, User } from '../models';
+import { ITeamResponse } from '../interfaces/team';
```
[the remark this answers is not in the document]
Comment (birdglove2) [reply users/src/controllers/team.controller.ts]: done

File: users/src/routes/__test__/user_test/user-info.test.ts [hunk 7]
```diff
@@ -1,17 +1,26 @@
 import request from 'supertest';
 import { app } from '../../../app';
-import { Member } from '../../../models/member.model';
+import { IsMember } from '../../../models/isMember.model';
```
[the remark this answers is not in the document]
Comment (birdglove2) [reply users/src/routes/__test__/user_test/user-info.test.ts]: done

**Pull request merged**